src/test/game.test.js

```diff
@@ -0,0 +1,6 @@
+import GameLogic from '../js/game';
```

Comment:
## Suggested requirements:
In our game logic, kindy let's add tests to check for: 👇 
- [ ] Draw.
- [ ] Game reset.
- [ ] Player input.
- [ ] Winning conditions.
- [ ] Invalid move by the player.